src/shared/ui/utils/money-formatters/money-formatters.test.ts

```diff
@@ -0,0 +1,11 @@
+import { expect, test } from 'vitest';
+import { numberFormatter } from './money-formatters';
+
+test.each([
+  [1234567.22312, '1,234,567.22312'],
+  [1234123, '1,234,123'],
+  [0.22312, '0.22312'],
+  [100.1, '100.1'],
+])('format number', (input, expected) => {
```

Comment:
```suggestion
])('format %f as %s', (input, expected) => {
```

Better test reports.